test/game.test.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+var expect = require('chai').expect;
+var Grid = require('../app/scripts/grid');
+var Game = require('../app/scripts/game');
+
+describe('Game', function () {
+
+  it('should determine a winner', function () {
+    var game = new Game({
+      grid: new Grid({
```

Comment:
Since 755308a5cafc1b53a590f46537a758f416079048, I made all parameters to the `Game` constructor completely optional, so you no longer need to pass in a new `Grid` object (it defaults to a 7-column, 6-row grid).